src/shared/components/mutationTable/column/clonal/ClonalColumnFormatter.spec.tsx

```diff
@@ -10,14 +10,17 @@ describe('ClonalColumnFormatter', () => {
         clonalElementProperties: any,
         expectedSampleId: string,
```

Comment:
* adding this comment in case we want all references of facets/FACETS replaced with "ascn/ASCN" 

found references to `FACETS`